crypto/fipsmodule/kem/internal.h

```diff
@@ -98,6 +98,17 @@ int KEM_KEY_set_raw_secret_key(KEM_KEY *key, const uint8_t *in);
 int KEM_KEY_set_raw_key(KEM_KEY *key, const uint8_t *in_public,
                                       const uint8_t *in_secret);
 
+// KEM_KEY_set_raw_keypair_from_seed function generates a keypair from the
+// given seed using the appropriate key generation function based on the
+// KEM variant, then allocates and sets both public and secret key buffers
+// within the given |key|.
+//
+// NOTE: The seed must be exactly 64 bytes for all ML-KEM variants.
+//       The caller must ensure the seed CBS contains valid data.
+//       |key->kem| must be initialized and |key->public_key| and 
+//       |key->secret_key| must both be NULL.
+int KEM_KEY_set_raw_keypair_from_seed(KEM_KEY *key, const CBS *seed);
```

Comment:
could we keep this module unaware of `CBS` stuff (all other functions here work with simple byte arrays)?

Reply:
My motivation was consistency with other functions in this header file. I was not aware of `PQDSA_KEY_set_raw_keypair_from_seed`. I think I'd like both of these to use raw byte buffers, but we can leave that for another time, not a blocker for this PR.